src/jv.h

```diff
@@ -16,18 +16,26 @@ typedef enum {
   JV_KIND_OBJECT
 } jv_kind;
 
+typedef enum {
+  JV_SUBKIND_NONE,
+  JV_SUBKIND_INT64,
+  JV_SUBKIND_UINT64,
+} jv_subkind;
+
 struct jv_refcnt;
 
 /* All of the fields of this struct are private.
    Really. Do not play with them. */
 typedef struct {
   unsigned char kind_flags;
-  unsigned char pad_;
+  unsigned char subkind_flags;
```

Comment:
Well, yes, we lose the padding.  I suppose I could encode sub-kind into kind_flags.  We could always do that later if ever we want to use the padding for something else.